Scala Language Exploration (Class A)

keenanleman opened the pull request:
Merubah bahasa Java ke bahasa Scala, kelas A

Commit:
Menambah folder A dan menjawab nomor 1 - 3 dan nomor 6

louisemarcho committed:
nomor 7 - 10 nih

Commit:
nomor 7 - 10 nih

KrisogonusFerdieRendragraha committed:
Ngerjakan no. 17 - 20 (Rendra)

KrisogonusFerdieRendragraha committed:
Ngerjakan no. 17 - 20 (Rendra)

BarsyaPrastoro committed:
Merge branch 'master' of https://github.com/keenanleman/LanguagesExploration

BarsyaPrastoro committed:
nambahin nomor 14, 15, 16.

TimusVPF committed:
Update LanguagesExploration.scala

Update numbers 21, 22, and 23.
Will be updated at a later date.

TimusVPF committed:
Update LanguagesExploration.scala

Fixing the tabbing. Converting tabs into spaces.

sembronaut committed:
Update LanguagesExploration.scala

Saya menambahkan nomor 11 sampai 13 yea

Commit:
Fixed: Nomor 2 dan 3

sembronaut committed:
Fixing number 12 to 13

BarsyaPrastoro committed:
number 14 and 16 fixed.

Commit:
Changed: Number 3

sembronaut committed:
Fixing number 12 to 13 (2)

Nomor 12 sama 13 gue bikin sebaris plus komennya juga

Commit:
Merge branch 'master' of https://github.com/keenanleman/LanguagesExploration

TimusVPF committed:
Update LanguagesExploration.scala

Adding number 21 and comments for most (if not all) subsections in the code.

Commit:
Merge branch 'master' of https://github.com/keenanleman/LanguagesExploration

Conflicts:
	A/LanguagesExploration.scala

louisemarcho committed:
Louise : edit nomor 7

TimusVPF committed:
Update No 22b

Update no 22, mengganti nama dari "setPrivateInt_=" menjadi setPrivateInt saja.

pascalalfadian commented:
Sudah bagus semua, kecuali di point 23:

``` scala
    // 23. Memastikan method overriding bekerja
    println("[26] publicSubClass.getPrivateInt() = " + publicClass.getPrivateInt())
```

Program kamu menghasilkan nilai 13 bukan 26. Mohon diperbaiki (bugfix nya cukup trivial).

Commit:
Fixed: Nomor 23

Pemanggilan method getPrivateInt pada objek publicClass, diganti menjadi pemanggilan method getPrivateInt pada objek publicSubClass

keenanleman commented:
baik pak, sudah di perbaiki.
